From: Tobias Hansen <thansen@debian.org>
Date: Sun, 12 Jun 2022 10:11:53 -0400
Subject: disable_tests

Descriptions: Disable some tests that are failing on certain architectures.
 The bugs have been reported upstream, see links in the patch below.
 numpy_subarray fails on big-endian architectures,
 numpy_memoryview on all powerpc architectures, arm64 and alpha,
 apparently due to a bug in numpy.
---
 tests/bugs.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tests/bugs.txt b/tests/bugs.txt
index e853b45..55f5804 100644
--- a/tests/bugs.txt
+++ b/tests/bugs.txt
@@ -55,3 +55,9 @@ pyregr.test_urllibnet
 
 # Inlined generators
 inlined_generator_expressions
+
+# Skipped in Debian
+# https://github.com/cython/cython/issues/2308
+numpy_memoryview
+# https://github.com/cython/cython/issues/1982
+numpy_subarray
